### DISCUSSION TITLE
How can js-cookie help Open Source funding?

### DISCUSSION THREAD

**Discussion by FagnerMartinsBrack** [js-cookie/js-cookie#755] [Ideas]
<sup>This is a continuation of [this thread](https://github.com/js-cookie/js-cookie/pull/744), which was the first attempt to help fund Open Source using crypto. However, it doesn't work for the reasons in the thread itself, so we're back to the drawing board.</sup>

I've read countless posts talking about how many Open Source devs work below the poverty line. The js-cookie project has millions of downloads on NPM per week, billions on JSDelivr, and thousands of unique visitors every day. How can we direct that traffic from js-cookie to help with Open Source funding? Perhaps just to create something at the top of the README to create awareness for those who see value in this library that Open Source is in crisis?

**Comment by FagnerMartinsBrack** [comment]
Some ideas so far to add to the README:

1. Add a link pointing to Open Collective.
2. Add a link to some specific Open Source projects that have the "Github Sponsors" or other kind of funding option in their project

Any other ideas will be much appreciated.

**Reply by carhartl** [reply]
Add a link pointing to

https://github.com/sponsors/community

**Comment by carhartl** [comment]
This would be useful: https://github.com/github/feedback/discussions/10177
